client/tailwind.config.js

```diff
@@ -3,20 +3,39 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
+      backgroundImage: (theme) => ({
+        'BlobL': "url('http://localhost:3000/assets/BlobL.svg')",
+        'BlobR': "url('http://localhost:3000/assets/BlobR.svg')",
+        'terms': "url('http://localhost:3000/assets/terms.svg')",
+        'shared_workspace': "url('http://localhost:3000/assets/shared_workspace.svg')",
+        'contact': "url('http://localhost:3000/assets/contact.svg')",
+      }),
       colors: {
       primary_bg_color: "#f4eeff", // main bg
       secondary_bg_color: "#a6b1e1", // secondary bg
       tertiary_bg_color: "#dcd6f7", // dropdown menu, tech stack tags
       primary_title_color: "#424874", // titles 
-      header_color: "#424874" // header
+      header_color: "#424874", // header
+      bolbR:"DCD6F7", // bolbR color
```

Comment:
```suggestion
      bolbR:"#DCD6F7", // bolbR color
```
Or not need this.  you can use `tertiary_bg_color` instead as this is the same colour.